### PR TITLE
Adding the maintainers approved in 2023Q1 Core Maintainers meeting

### PR DESCRIPTION
**Pull request by gottbrath**
Added Nikita to Core Maintainers
Merged MKLDNN with CPU Performance
Renamed CUDA to GPU Performance
Added Jiong to Compiler and CPU Performance
Added Xiaobing to CPU Performance
Marking Vitaly and Jian Hui as Emeritus

### CODE REVIEW

**Review by dzhulgakov — COMMENTED** [review]
Thanks! Should we update https://github.com/pytorch/pytorch/blob/master/.github/merge_rules.yaml in the same go?

File: docs/source/community/persons_of_interest.rst [hunk 1]
```diff
@@ -138,17 +140,22 @@ Fast Fourier Transform (torch.fft)
...
 -  (emeritus) Sam Gross (`colesbury <https://github.com/colesbury>`__)
 -  (emeritus) Christian Puhrsch (`cpuhrsch <https://github.com/cpuhrsch>`__)
 -  (emeritus) Ilia Cherniavskii (`ilia-cher <https://github.com/ilia-cher>`__)
+-  (emeritus) Junjie Bai (`bddppq <https://github.com/bddppq>`__)
+-  (emeritus) Yinghai Lu (`yinghai <https://github.com/yinghai>`__)
+-  (emeritus) Vitaly Fedyunin (`VitalyFedyunin <https://github.com/VitalyFedyunin>`__)
+-  (emeritus) Jianhui Li (`Jianhui-Li <https://github.com/Jianhui-Li>`__)
 
-NVIDIA / CUDA
+GPU Performance (Torch Inductor / Triton / NVIDIA / CUDA )
```
[the remark this answers is not in the document]
Comment (gottbrath) [reply docs/source/community/persons_of_interest.rst:158]: In that same line of thinking should put the ROCm maintainers in this group as well?

**Comment by gottbrath** [comment]
> Thanks! Should we update https://github.com/pytorch/pytorch/blob/master/.github/merge_rules.yaml in the same go?

Good catch. Yes probably. I am not as familiar with that, but it looks fairly straightforward. Let me give it a stab.

**Comment by gottbrath** [comment]
Just tagging that Mike Brown noticed that the Readme.md file had some out of date info in https://github.com/pytorch/pytorch/issues/98456

**Review by ezyang — APPROVED** [review]

**Comment by gottbrath** [comment]
made some updates. 

there is still some rationalization that we need to do around

Module structure (we should probably merge the rocm in with the Compiler GPU backend ) 
module level structure should be aligned with the yaml file
The thanks list in the readme.md seems quite out of date.

**Review by soumith — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -435,7 +435,7 @@ To learn more about making a contribution to Pytorch, please see our [Contributi
 
 PyTorch is a community-driven project with several skillful engineers and researchers contributing to it.
 
-PyTorch is currently maintained by [Adam Paszke](https://apaszke.github.io/), [Sam Gross](https://github.com/colesbury), [Soumith Chintala](http://soumith.ch) and [Gregory Chanan](https://github.com/gchanan) with major contributions coming from hundreds of talented individuals in various forms and means.
+PyTorch is currently maintained by [Soumith Chintala](http://soumith.ch), [Gregory Chanan](https://github.com/gchanan), [Dmytro Dzhulgakov](https://github.com/dzhulgakov), [Edward Yang](https://github.com/ezyang), and [Nikita Shulga](https://github.com/malfet) with major contributions coming from hundreds of talented individuals in various forms and means.
 A non-exhaustive but growing list needs to mention: Trevor Killeen, Sasank Chilamkurthy, Sergey Zagoruyko, Adam Lerer, Francisco Massa, Alykhan Tejani, Luca Antiga, Alban Desmaison, Andreas Koepf, James Bradbury, Zeming Lin, Yuandong Tian, Guillaume Lample, Marat Dukhan, Natalia Gimelshein, Christian Sarofeen, Martin Raison, Edward Yang, Zachary Devito.
```
Comment (dzhulgakov) [inline README.md:439]: Hm, it's indeed quite out of date. Maybe link to PT paper (https://arxiv.org/pdf/1912.01703.pdf) it has the original list of acknowledgments and persons_of_interest.rst for module-level

File: docs/source/community/persons_of_interest.rst [hunk 3]
```diff
@@ -138,18 +140,23 @@ Fast Fourier Transform (torch.fft)
...
 
-CPU Performance / SIMD
-~~~~~~~~~~~~~~~~~~~~~~
+CPU Performance (Torch Inductor / MKLDNN)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  Vitaly Fedyunin (`VitalyFedyunin <https://github.com/VitalyFedyunin>`__)
 -  Mingfei Ma (`mingfeima <https://github.com/mingfeima>`__)
+-  Jiong Gong (`jgong5 <https://github.com/jgong5>`__)
+-  Xiaobing Zhang (`XiaobingSuper <https://github.com/XiaobingSuper>`__)
```
Comment (dzhulgakov) [inline docs/source/community/persons_of_interest.rst:148]: looks like these folks were already in merge_rules.yaml's sections of "CPU ATen backend" and "CPU frontend", so they have merge access already :)

**Review by dzhulgakov — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (gottbrath) [reply README.md:439]: I do think we should do that in a follow up PR.

**Comment by gottbrath** [comment]
ok, I don't think this needs an additional release note. So while this is user facing I'll go ahead and add the not user facing tag.

**Pull request closed without merging**